root/static/scripts/edit/check-duplicates.js

```diff
@@ -115,12 +115,17 @@ function isPlaceCommentRequired(duplicates) {
   });
 }
 
-function isCommentRequired(type, duplicates) {
+function isCommentRequired(type, name, duplicates) {
   if (type === 'place') {
     return isPlaceCommentRequired(duplicates);
   }
 
-  return duplicates.length > 0;
+  for (let duplicate of duplicates) {
```

Comment:
I remember `let ... of ...` not working before, it needed `Symbol` to be defined or something. Can we replace with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?